src/test/java/rx/CompletableTest.java

```diff
@@ -2605,44 +2605,44 @@ public void call() {
     
     @Test(expected = NullPointerException.class)
     public void subscribeTwoCallbacksFirstNull() {
-        normal.completable.subscribe(null, new Action0() {
+        normal.completable.subscribe(null, new Action1<Throwable>() {
             @Override
-            public void call() { }
+            public void call(Throwable throwable) {}
         });
     }
     
     @Test(expected = NullPointerException.class)
     public void subscribeTwoCallbacksSecondNull() {
-        normal.completable.subscribe(null, new Action0() {
```

Comment:
Didn't catch that on initial PR, test was obviously copied from `subscribeTwoCallbacksFirstNull` and wasn't changed, second parameter should have been `null`.